projects/configs/tag-operations/xray.yaml

```diff
@@ -0,0 +1,86 @@
+#  Copyright (c) University College London Hospitals NHS Foundation Trust
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+# Default configuration for base with commented links to their respective image modalities
+
+
+
+- name: "Volumetric Properties"
+  group: 0x0008
+  element: 0x9206
+  op: "keep"
```

Comment:
This one, and perhaps the next couple looks like they are in enhanced MRs, which we want to also support so I think this should also be added the the MRI.yaml as well as being in x-ray

Reply:
Volumetric Properties and Volume Based Calculation Technique belong to both mri and xray, therefore they are already in both [mri](https://github.com/SAFEHR-data/PIXL/blob/482-configs-for-default-de-identification-of-ion-dri-projects/projects/configs/tag-operations/mri.yaml) and [x-ray](https://github.com/SAFEHR-data/PIXL/blob/482-configs-for-default-de-identification-of-ion-dri-projects/projects/configs/tag-operations/xray.yaml)